types/serde_test.go

```diff
@@ -0,0 +1,33 @@
+package types
+
+import (
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestTextMarshalling(t *testing.T) {
+	const destString = "0x000000000000000000000000f5a1bb5607c9d079e46d1b3dc33f257d937b43bd"
+	dest := Destination(common.HexToHash("0x000000000000000000000000f5a1bb5607c9d079e46d1b3dc33f257d937b43bd"))
```

Comment:
I'm confused about this syntax -- is `Destination` a function? Or is it a different way of creating a `Destination` value?

Reply:
ah, it is type conversion...